### ISSUE TITLE
combination horizons described with 'and'

### ISSUE THREAD

**Issue by dylanbeaudette** [dylanbeaudette/parse-osd#9]
See [Leon](https://casoilresource.lawr.ucdavis.edu/sde/?series=leon) as an example.

Double-check: does it make sense to convert "X and Y" to "X/Y".

**Comment by brownag** [comment]
I took a peek at the Leon OSD. I do think it makes sense to convert  "and" -> "/"

But a wrinkle with handling the parsing of combination horizons is that there are likely to be descriptors unique to each part. 

For instance: 

> Bw and Bh--22 to 25 inches; 80 percent (Bw) dark yellowish brown (10YR 4/4) and 20 percent (Bh) dark brown (10YR 3/3) sand; very weak medium and coarse subangular blocky structure; very friable; common fine and medium pores; very strongly acid; clear wavy boundary. (0 to 15 inches thick)

Currently parsed OSD will contain "Bh" as the designation with 10YR 4/4 as the color--which sort of mixes things up. The dominant part (80%) is Bw, not Bh. The dominant color is 10YR 4/4 for the Bw.

**Comment by dylanbeaudette** [comment]
Yes and of course this is complicated. Long-term the OSDs will come from a real pedon object. We are about 1-2 years from that so we need an approximate solution until then.

  * Lets move this issue to SKB, I'd like to retire this repo soon.
  * Flag all OSDs that contain "and" in the horizon designation as part of the weekly QC report.
  * I'll talk with SRSS about requesting a "and" → "/" change as part of the OSD fixup project.
  * Lets schedule some time / tests for better matching of multiple colors per horizon. The existing code is close, but not enough logic / flexibility to work in general.
